doc/development/source/tutorial-settings.rst

```diff
@@ -130,6 +130,31 @@ Well-behaved widgets remember their settings - the state of their
 checkboxes and radio-buttons, the text in their line edits, the
 selections in their combo boxes and similar.
 
+Persisting defaults
+*******************
+Default setting values are not set in stone. As different users might
+prefer different defaults, Orange remembers which settings were used
+and stores them as the new defaults when the widget is deleted. Widgets
+removed from there on will start with the new defaults.
+
+Updated defaults are stored in user's profile based on the operation system.
```

Comment:
operation -> operating.

Perhaps better: *Updated defaults are stored in user's profile. It's location depends on the operating system:*